src/main/scala/metadata.scala

```diff
@@ -0,0 +1,83 @@
+package suffuse
+
+/** AKey is an attribute key: a Metadata map key which carries
+ *  both the type of the corresponding value, and a value to
+ *  provide when there is no value in the map. This as opposed
+ *  to wrapping everything in sight in Option.
+ */
+final class AKey[A](description: String, empty: => A) {
+  def emptyValue: A = empty
```

Comment:
What is "not supplying stats in getattr"? There is certainly something in that memory. Either the fuse-jna layer is injecting default values, or the memory is zeroed out and remains zero, or... but in every case a value to represent emptiness HAS been chosen. Only, not by us.